app/controllers/categories_controller.rb

```diff
@@ -163,6 +163,7 @@ def update
         end
       end
 
+      DiscourseEvent.trigger(:category_updated, result)
```

Comment:
The value of `result` is just `true` or `false` right? I feel like the event would probably want a reference to the category itself and should only happen in the `if result = ` block. Does that make sense?